### PR TITLE
Update Project Profile Adding Nilakshi Sahai

### LINKED ISSUE #7648: Update Project Profile: LA TDM Calculator Add Nilakshi Sahai
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/tdm-calculator.md` file.
- [x] Observe the existing syntax of the front matter block [^1] in the file.
- [x] Find the `leadership` variable and add the following profile. 
```
  - name: Nilakshi Sahai
    github-handle: NilakshiS
    role: UX Design Lead
    links:
      slack: https://hackforla.slack.com/team/U06SYVD69L7
      github: https://github.com/NilakshiS
    picture: https://avatars.githubusercontent.com/NilakshiS
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] LA TDM Calculator page [^2]
- [x] Once your pull request is merged, go to the initiating ER [^3]
  - [ ] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move initiating ER [^3] to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        
      </details>

### Resources/Instructions
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/tdm-calculator
[^3]: initiating ER:  #7305

### PR DESCRIPTION
**Pull request by belunatic**
Fixes #7648

### What changes did you make?
  - I added Nilakshi Sahai to the current Project Team Section

### Why did you make the changes (we will use this info to test)?
  
  - To update the Current Project Team Members

<h3>CodeQL Alerts</h3>

After the PR has been submitted and the resulting GitHub actions/checks have been completed, developers should check the PR for CodeQL alert annotations.

<details><summary>Check the PR's comments. If present on your PR, the CodeQL alert looks similar as shown</summary>
  
</details>

Please let us know that you have checked for CodeQL alerts. **Please do not dismiss alerts.**
- [x] I have checked this PR for CodeQL alerts and none were found.
- [ ] I found CodeQL alert(s), and (select one):
   - [ ] I have resolved the CodeQL alert(s) as noted
   - [ ] I believe the CodeQL alert(s) is a false positive (Merge Team will evaluate)
   - [ ] I have followed the Instructions below, but I am still stuck (Merge Team will evaluate)

<details><summary>Instructions for resolving CodeQL alerts</summary>

If CodeQL alert/annotations appear, refer to [How to Resolve CodeQL alerts](https://github.com/hackforla/website/issues/6463#issuecomment-2002573270).  

In general, CodeQL alerts should be resolved prior to PR reviews and merging

</details>

### Screenshots of Proposed Changes To The Website (if any, please do not include screenshots of code changes)
 

<details>
<summary>Visuals before changes are applied</summary>

FireShot Capture 070 - LA TDM Calculator - Hack for LA - localhost

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
FireShot Capture 071 - LA TDM Calculator - Hack for LA - localhost

</details>

### CODE REVIEW

**Review by codyyjxn — CHANGES_REQUESTED** [review]
@belunatic thank you for taking on this issue.
## Suggestions 
- You need to respond to the bot in the comments below on the original issue. 
- You need to fix the conflict

**Comment by belunatic** [comment]
updated image after resolving the conflict

FireShot Capture 072 - LA TDM Calculator - Hack for LA - localhost

**Review by codyyjxn — DISMISSED** [review]
@belunatic  Great job fixing the merge conflict! Thank you for your contribution keep it up !

**Comment by belunatic** [comment]
Thank you @codyyjxn, I appreciate you!

**Comment by jphamtv** [comment]
Hi @belunatic - Please let team members self-assign PR reviews. Available reviewers will pick up PRs from the queue when they can work on them. This helps us maintain an efficient review process where available reviewers can schedule their workloads. Thanks!

**Comment by belunatic** [comment]
@jphamtv I appreciate you! Have a great weekend!

File: _projects/tdm-calculator.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ leadership:
       slack: 'https://hackforla.slack.com/team/U01RDSRT46R'
       github: 'https://github.com/Jane4925'
     picture: https://avatars.githubusercontent.com/Jane4925
+  - name: Nilakshi Sahai
+    github-handle: NilakshiS
+    role: Lead UX Design Lead
```
Comment (ExperimentsInHonesty) [inline _projects/tdm-calculator.md]:
```suggestion
    role: UX Design Lead
```

**Review by ExperimentsInHonesty — DISMISSED** [review]
This looks good.  Its ready for merge.  I just made one change to her title.  Changed it 
from: Lead UX Design Lead
to: UX Design Lead

**Comment by ExperimentsInHonesty** [comment]
I am merging this to reduce confusion, since it already had two reviews.

**Pull request merged**